hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/ScheduleUtility.java

```diff
@@ -314,7 +314,7 @@ public static String calculateStringHash(final @NonNull Schedule scheduleToHash)
         }
         if (scheduleToHash.providedExpirationSecond() != Schedule.DEFAULT.providedExpirationSecond()) {
             // @todo('7905') fix to not use Proto serialization
-            //            hasher.putLong(scheduleToHash.providedExpirationSecond());
+            // hasher.putLong(scheduleToHash.providedExpirationSecond())
```

Comment:
This class is renamed and reworked in another PR, which makes this change unnecessary.

Reply:
Thanks for the heads-up, but I will merge it since I have the approval. It shouldn't be a problem for the other PR.